src/ExcelIO/ExcelIOBase.cs

```diff
@@ -217,6 +217,21 @@ protected static string GetColumnLetterIDsOfColumnNames(ref SpreadsheetDocument
         #endregion
 
         #region CheckExists
+        /// <summary>
+        /// Check if a worksheet does exist in a spreadsheet and optional it returns the sheet(s) in (parameter) 'sheetsIEnum'.
+        /// </summary>
+        /// <param name="spreadsheetDocument">
+        /// Spreadsheet where to search for the worksheet.
+        /// </param>
+        /// <param name="worksheetName">
+        /// Name of the worksheet that should be searched.
+        /// </param>
+        /// <param name="sheetsIEnum">
+        /// This returns the sheet(s) that do have the name of (parameter) 'worksheetName'.
+        /// </param>
+        /// <returns>
+        /// True, if worksheet with (parameter) 'worksheetName' does exist, otherwise False.
+        /// </returns>
         public static bool WorksheetExists(ref SpreadsheetDocument spreadsheetDocument, string worksheetName, out IEnumerable<Sheet> sheetsIEnum)
```

Comment:
Wieso wird SpreadsheetDocument eigentlich als ref übergeben?

Reply:
@jonashoelzel Aus demselben Grund, weshalb die meisten Parameter mit ref übergeben werden: Um zu verdeutlichen das die Funktion Veränderungen am Spreadsheet durchführt. Prinzipiell ist das ref unnötig, da sowieso die Referenz übergeben wird. Aber durch das ref soll noch mal daran erinnert werden, dass das Spreadsheet in der Funktion manipuliert wird, anstatt einfach nur verwendet (gelesen) zu werden.

Reply:
Aber diese Erklärung würde ich auch später in die Docs packen. Nur sind die Docs noch nicht so weit.

Reply:
Das ist nur wegen dem einheitlichen Design. Das hat eigentlich keinen tieferen Sinn, außer das dann immer, wenn ein Spreadsheet übergeben wird es als ref übergeben wird.